dts/arm/st/n6/stm32n6.dtsi

```diff
@@ -568,6 +568,66 @@
 			status = "disabled";
 		};
 
+		spi1: spi@52003000 {
+			compatible = "st,stm32h7-spi", "st,stm32-spi-fifo", "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x52003000 0x400>;
+			interrupts = <153 0>;
+			clocks = <&rcc STM32_CLOCK(APB2, 12)>;
+			status = "disabled";
+		};
+
+		spi2: spi@50003800 {
+			compatible = "st,stm32h7-spi", "st,stm32-spi-fifo", "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x50003800 0x400>;
+			interrupts = <154 0>;
+			clocks = <&rcc STM32_CLOCK(APB1, 14)>;
+			status = "disabled";
+		};
+
+		spi3: spi@50003c00 {
+			compatible = "st,stm32h7-spi", "st,stm32-spi-fifo", "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x50003C00 0x400>;
```

Comment:
reg = <0x50003C00 0x400>;
```suggestion
			reg = <0x50003c00 0x400>;
```

Reply:
There are other cases like this in the dtsi. I can do a full pass later and fix them all at once. I'd prefer to have this PR merged fast before the feature freeze.